cluster/log-dump/log-dump.sh

```diff
@@ -41,7 +41,7 @@ readonly master_ssh_supported_providers="gce aws"
 readonly node_ssh_supported_providers="gce gke aws"
 readonly gcloud_supported_providers="gce gke"
 
-readonly master_logfiles="kube-apiserver.log kube-apiserver-audit.log kube-scheduler.log kube-controller-manager.log etcd.log etcd-events.log glbc.log cloud-controller-manager.log cluster-autoscaler.log kube-addon-manager.log konnectivity-server.log fluentd.log kubelet.cov"
```

Comment:
we loose this log file?

Reply:
No, it was something that I needed to clean up. Thanks for pointing out.